mmdet/models/dense_heads/solo_head.py

```diff
@@ -135,13 +135,18 @@ def resize_feats(self, feats):
         for i in range(len(feats)):
             if i == 0:
                 out.append(
-                    F.interpolate(feats[0], scale_factor=0.5, mode='bilinear'))
+                    F.interpolate(
+                        feats[0],
+                        size=feats[i + 1].shape[-2:],
+                        mode='bilinear',
+                        align_corners=False))
             elif i == len(feats) - 1:
                 out.append(
                     F.interpolate(
                         feats[i],
                         size=feats[i - 1].shape[-2:],
-                        mode='bilinear'))
+                        mode='bilinear',
+                        align_corners=False))
```

Comment:
why delete this line?

Reply:
I just add `align_corners=False` arg to suppress PyTorch's warning and `align_corners=False` is exactly the default behavior when this arg is not provided. 
But if we forgot to add this line, PyTorch would warn about the default behavior change at some old torch version.